examples/env-vars/astro.config.mjs

```diff
@@ -2,5 +2,5 @@ import { defineConfig } from 'astro/config';
 
 // https://astro.build/config
 export default defineConfig({
-	renderers: [],
+	integrations: [],
 });
```

Comment:
```js
export default defineConfig({})
```

Nit: to match other suggestions, just collapse config when integrations aren't needed. Weirdly don't have a "suggest commit" button while reviewing so use that code snippet above 🙃